tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -4194,10 +4194,10 @@ Status MklLayoutRewritePass::Run(const GraphOptimizationPassOptions& options) {
   if (options.graph == nullptr && options.partition_graphs == nullptr) {
     return OkStatus();
   }
-  if (!IsMKLEnabled()) {
-    VLOG(2) << "TF-MKL: MKL is not enabled";
-    return OkStatus();
-  }
+
+  if (!IsOneDNNEnabled()) {
+    VLOG(2) << "TF-MKL: oneDNN is not enabled";
+    return Status::OK();
```

Comment:
I think we are missing a closing bracket (`}`).

Reply:
Must be introduced when addressing merge conflicts (from other oneDNN PRs). Fixed.

Thanks!